### DISCUSSION TITLE
tostky-plugin-no-longer-working :(

### DISCUSSION THREAD

**Discussion by Geppie-99** [runelite/runelite#18350] [Q&A] [answered]
Hey everybody,

Since the last update (September 25, 2024) unfortunately this plugin does not seem to work anymore.
It has disappeared from the sidebar I still see it in my .runelite folder and also it is still in settings of runelite.
Could someone please check what exactly is going on?

Thanks in advance

Geppie

https://github.com/example-git/tostky-plugin

**Comment by cdfisher** [comment]
It's very possible that something with the latest game update broke the plugin. We don't provide support for Plugin Hub plugins, but if you post your `client.log` and `launcher.log`, I can take a look. Instructions on how to get to your logs can be found [here](https://github.com/runelite/runelite/wiki/Troubleshooting-problems-with-the-client).

Is the plugin enabled for your current profile?

**Reply by Geppie-99** [reply]
Thanks for your quick reply, if you could do that would be great!

Please see below my logs.

[client.log](https://github.com/user-attachments/files/17174887/client.log)
[launcher.log](https://github.com/user-attachments/files/17174888/launcher.log)

**Reply by cdfisher** [reply]
`ERROR n.r.client.plugins.PluginManager - Unable to start plugin TostkyPlugin
net.runelite.client.plugins.PluginInstantiationException: java.lang.IllegalStateException: must be called on client thread`

It looks like the plugin is either broken or conflicting with something, and is.as a result failing to start. You can open an issue about the plugin failing to start [here](https://github.com/example-git/tostky-plugin/issues).

**Reply by Geppie-99** [reply]
Thank you, I got the following error when I try to submit a issue xd
Pull request creation failed. Validation failed: must be a collaborator

**Reply by cdfisher** [reply, marked as answer]
It looks like a fix for this just got merged so I would guess that this problem should be resolved for you within the next few minutes if it's not already.
